### PR TITLE
Added integration tests to appengine-push.

### PR DESCRIPTION
**Pull request by tmatsuo**
I'm going to add real integration tests to this PR.

### CODE REVIEW

File: appengine-push/test_deploy.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        """Test accessing the top page."""
+        self.conn.request('GET', '/')
+        resp = self.conn.getresponse()
+        # This ensures that our App Engine service account is working
+        # correctly.
+        self.assertEquals(200, resp.status)
+
+    def fetch_messages(self):
+        """Fetch messages"""
+        conn = httplib.HTTPSConnection(get_gae_host())
```
Comment (jonparrott) [inline appengine-push/test_deploy.py]: why httplib instead of httplib2?

**Comment by jonparrott** [comment]
LGTM with small nit.

File: appengine-push/test_deploy.py (hunk 1 above)
Comment (tmatsuo) [inline appengine-push/test_deploy.py]: No strong readon, but httplib is in the standard lib and it satisfy my needs here.

**Comment by tmatsuo** [comment]
Thanks, I changed to httplib2 from httplib.
Also added some integration tests for checking consistency of base64 variants used on the server side and the client side.

@jonparrott PTAL

**Comment by jonparrott** [comment]
LGTM

**Comment by tmatsuo** [comment]
Thanks! will merge when the travis says OK :)

**Pull request merged**